### DISCUSSION TITLE
linear-gradient in Daily Target (18/09/2023)

### DISCUSSION THREAD

**Discussion by FranPetkovic** [cssbattle/forum#359] [General]
I am using this linear gradient linear-gradient(90deg,#802471 25%,#4F0843 75%), but getting only 99.8%

**Comment by PavButsa** [comment]
You can improve to 99.9% match if you change 25% to 25.5% but nobody got 100% match by now. I think this target is bugged.

![css_battle_2023_09_18_success_rate_0](https://github.com/cssbattle/forum/assets/8935771/1cfe3b03-835c-49e3-9ca3-848ed0ecd854)
